Amended Virtual Event API Docs (#21541)
microsoftgraph/microsoft-graph-docs: api-reference/beta/api/virtualeventregistrant-get.md

|Delegated (personal Microsoft account)|Not supported.|
|Application|VirtualEvent.Read.All|

> [!NOTE]
>

> > To use application permission for this API, tenant administrators must create an [application access policy](/graph/cloud-communication-online-meeting-application-access-policy) and grant it to a user to authorize the app configured in the policy to get registrants of a virtual event created by that user.

Reviewer: ```suggestion
> To use application permissions for this API, tenant administrators must create an [application access policy](/graph/cloud-communication-online-meeting-application-access-policy) and assign it to a user.The authorized application will access registrants' information from virtual events created by that specific user.
```
